bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -171,6 +171,24 @@ def row_key(self):
         """
         return self._row_key
 
+    def cell_value(self, column_family_id, column, index=0):
+        """Get a cell value
+
+        :type column_family_id: str
+        :param column_family_id: The ID of the column family. Must be of the
+                                 form ``[_a-zA-Z0-9][-_.a-zA-Z0-9]*``.
+        :type column: bytes
+        :param column: The column within the column family where the cell
+                       is located.
+
+        :type index: int
+        :param index: The offset in the series of values, default = 0
+
+        :rtype:  bytes or :class:`int`
```

Comment:
should this be bytes only?

Reply:
This documentation was copied verbatim from method set_cell on the different row classes in row.py.  Any changes here should be reviewed in the context of all the documentation.

Thanks!

Reply:
Hello @zakons, my apologies.  I'm relatively new here and I didn't mean to be offensive.  I wasn't sure of the answers myself and therefore phrased it as a question.  Sorry if it seemed offensive.

Reply:
Actually, I agree it should be bytes only, but as I said, I copied the documentation for consistency.  I would encourage it be corrected and made consistent if this is correct. I will follow up on this.

Reply:
@zakons The distinction as that `set_cell()` will accept an integer and then encode the integer as a big-endian `long long` (`>q`) using the [`struct` library][1].

[1]: https://docs.python.org/3/library/struct.html